global.ini

```diff
@@ -320,7 +320,7 @@ BrightSky_Objective_Marker=Schiffskiosk
 BrightSky_Objective_Short=Ticket abgeben.
 BrightSky_TicketsFoundNotification=Neues Ticket wurde von ~mission(PlayerThatFound) gefunden
 BrightSky_Title=Ein Spießrutenlauf
-BrightSky_Unclaimed_Objective_Long=Holen Sie sich das nicht abgeholte Ticket, um es in den Kiosk zu verwandeln.
+BrightSky_Unclaimed_Objective_Long=Holen Sie sich das nicht abgeholte Ticket, um es in den Kiosk zu verwenden.
```

Comment:
Zeile komplett in: `BrightSky_Unclaimed_Objective_Long=Hole das unbeanspruchte Ticket ab, um es am Kiosk einzulösen.` ändern